addons/custom-fonts/addon.json

```diff
@@ -35,42 +35,114 @@
       "id": "navfnt",
       "type": "datalist",
       "default": "Helvetica",
-      "items": ["Sora", "Helvetica", "Arial", "Roboto", "Lucida Grande", "Verdana", "DejaVu Sans"]
+      "items": [
+        "Sora",
```

Comment:
Sora probably shouldn't be first, that was my fault.

Reply:
Yes. It's not a very legible font. Sora should be last.

Reply:
Fixed it. I did not put it last, but right along with the other fonts that are somewhat unreadable.